### PR TITLE
Update helm template usage

### PR DESCRIPTION
**Pull request by prachirp**
Clarify that chart name is optional and will have a default value of RELEASE-NAME if none is provided.

### CODE REVIEW

File: functions/ts/src/helm_template.ts [hunk 1]
```diff
@@ -83,11 +83,11 @@ Render chart templates locally using helm template. If input a list of configs i
 addition to arguments will overwrite any chart objects that already exist in the list.
 
 Configured using a ConfigMap with keys for ${CHART_NAME}, ${CHART_PATH}.
-Works with arbitrary helm template flags like --values:
+Works with arbitrary helm template flags like ${VALUES}:
 
-${CHART_NAME}: Name of helm chart.
+${CHART_NAME}: Name of helm chart. Default RELEASE-NAME.
```
Comment (frankfarzan) [inline functions/ts/src/helm_template.ts:88]: What's the significance of `RELEASE-NAME`, where is it defined/consumed? Maybe link to helm docs.

Comment (prachirp) [reply functions/ts/src/helm_template.ts:88]: This is not documented anywhere in helm docs.

[This doc](https://helm.sh/docs/topics/v2_v3_migration/#overview-of-helm-3-changes) says helm "requires release name or --generate-name argument" but this is not accurate as helm uses "RELEASE-NAME" as a default if the name is not provided.

Comment (frankfarzan) [reply functions/ts/src/helm_template.ts:88]: Let me rephrase. If someone just provides CHAR_PATH, what's the expected behavior? In which situation the user needs to explicitly provide the CHART_NAME?

Comment (prachirp) [reply functions/ts/src/helm_template.ts:88]: If someone just provides chart-path then helm template will output expanded configs that use the name RELEASE-NAME. The user should provide a name or use the --generate-name flag if they do not want to use the default chart name.

File: functions/ts/src/helm_template.ts [hunk 2]
```diff
@@ -83,11 +83,11 @@ Render chart templates locally using helm template. If input a list of configs i
...
 ${CHART_PATH}: Chart templates directory.
-${VALUES_PATH}: [Optional] Path to values file.
+${VALUES}: [Optional] Path to values file.
```
Comment (frankfarzan) [inline functions/ts/src/helm_template.ts:90]: nit: Consistent use of `[optional]`
